etc/kapacitor/kapacitor.conf

```diff
@@ -83,6 +83,12 @@ data_dir = "/var/lib/kapacitor"
   # This is insecure.
   insecure-skip-verify = false
 
+  # Host part of a bind addres for UDP listeners.
+  # For example if a UDP listener is using port 1234
+  # and `udp-bind = "hostname_or_ip"`,
+  # then the UDP port will be bound to `hostname_or_ip:1234`
+  # The default empty value will bind to all addresses.
+  udp-bind = ""
```

Comment:
@rossmcdonald Since I was in this code today, I added this option so that the UDP bind address is configurable separate from the `hostname` config option. Now the default is to bind to all addresses, which should work well.